osu.Game/Beatmaps/Formats/LegacyStoryboardDecoder.cs

```diff
@@ -77,12 +78,13 @@ private void handleGeneral(Storyboard storyboard, string line)
             }
         }
 
-        private void handleEvents(string line)
+        private void handleEvents(ReadOnlySpan<char> line)
         {
             decodeVariables(ref line);
 
             int depth = 0;
 
+            // TODO: Use IndexOfAnyExcept
```

Comment:
You know, with that method name, I'd rather *not* use whatever `IndexOfAnyExcept()` is. My head hurts trying to parse that.

Reply:
There are a lot of IndexOf* methods in BCL that get heavily SIMD optimized. The method name describes the SIMD operation.
I agree that a method like `TrimAny` can be better.